apps/complete_registration/src/screens/form/FormInput.tsx

```diff
@@ -34,7 +35,7 @@ const TextInput: FieldComponentType<
     FieldComponentProps & {
         inputClassName?: string
     }
-> = ({ label, mapping, inputClassName, className, ...rest }) => {
+> = ({ label, mapping, readOnly, inputClassName, className, ...rest }) => {
```

Comment:
`readonly` is a good addition! To be consistent we just need to add it to the other inputs too, at least TextAreaInput and Switch need to have readOnly since these three are the most used atm

Reply:
Let's add it in the future